src/NodeApi/Runtime/NativeLibrary.cs

```diff
@@ -68,8 +126,20 @@ public static nint GetExport(nint handle, string name)
     public static bool TryGetExport(nint handle, string name, out nint procAddress)
```

Comment:
Can we implement the `GetExport` in terms of the `TryGetExport`? Or we loose ability the for the error reporting?

Reply:
Implemented GetSymbol and pointed there. This logic duplicates dotnet/runtime.